_includes/user-menu.html

```diff
@@ -38,12 +38,14 @@
 			<li><a href="{{docu}}/administration/runtime.html">Runtime Commands</a></li>
 		</ul>
 	</li>
-	<li><a href="{{docu}}/features/index.html">Features</a>
+	<li><a href="{{docu}}/features/index.html">Basic Concepts</a>
 		<ul>
             <li><a href="{{docu}}/features/index.html">Overview</a></li>
+            <li><a href="{{docu}}/features/bindings-addons.html">Bindings and Addons</a></li>
+            <li><a href="{{docu}}/features/things-items.html">Things and Items</a></li>
             <li><a href="{{docu}}/features/sitemap.html">Sitemaps</a></li>
             <li><a href="{{docu}}/features/persistence.html">Persistence</a></li>
-            <li><a href="{{docu}}/features/automation/ruledsl.html">Automation</a>
+            <li><a href="{{docu}}/features/automation/ruledsl.html">Rules and Scripting</a>
```

Comment:
I think I was unclear here. Please do not change the other elements for now. Another person is working on redefining the structure.